src/Microsoft.ML.Data/DataView/CompositeRowToRowMapper.cs

```diff
@@ -19,6 +19,9 @@ public sealed class CompositeRowToRowMapper : IRowToRowMapper
         public Schema InputSchema { get; }
         public Schema Schema { get; }
 
+        [BestFriend]
+        internal IRowToRowMapper[] InnerMappers => _innerMappers;
```

Comment:
>InnerMappers [](start = 35, length = 12)

Just to re-iterate, get rid of `_innerMappers` and replace with autoimplemented property, e.g., `internal IRowToRowMapper[] InnerMappers { get; }`.